app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationViewModel.kt

```diff
@@ -115,20 +123,45 @@ class ConversationViewModel(
     val blindedRecipient: Recipient?
         get() = _recipient.value?.let { recipient ->
             when {
-                recipient.isOpenGroupOutboxRecipient -> recipient
-                recipient.isOpenGroupInboxRecipient -> repository.maybeGetBlindedRecipient(recipient)
+                recipient.isCommunityOutboxRecipient -> recipient
+                recipient.isCommunityInboxRecipient -> repository.maybeGetBlindedRecipient(recipient)
                 else -> null
             }
         }
 
-    private var communityWriteAccessJob: Job? = null
+    /**
+     * The admin who invites us to this group(v2) conversation.
+     *
+     * null if this convo is not a group(v2) conversation, or error getting the info
+     */
+    val invitingAdmin: Recipient?
+        get() {
+            val recipient = recipient ?: return null
+            if (!recipient.isGroupV2Recipient) return null
+
+            return repository.getInvitingAdmin(threadId)
+        }
 
     private var _openGroup: RetrieveOnce<OpenGroup> = RetrieveOnce {
         storage.getOpenGroup(threadId)
     }
     val openGroup: OpenGroup?
         get() = _openGroup.value
 
+    private val closedGroupMembers: List<GroupMember>
+        get() {
+            val recipient = recipient ?: return emptyList()
+            if (!recipient.isGroupV2Recipient) return emptyList()
```

Comment:
The property is called `closedGroupMembers` but it seems to check for groupV2 only. Is it a groupsV2 only thing?
Should it be renamed groupV2Members?

Reply:
Removed due to not in used